src/ansys/aedt/core/application/analysis.py

```diff
@@ -233,7 +233,7 @@ def materials(self):
         return self._materials
 
     @property
-    def setups(self):
+    def setups(self) -> list[Setup]:
```

Comment:
```suggestion
    def setups(self) -> List[Setup]:
```
I think you need to import List from typing and change `list` into `List`.
Once we move to Python3.9+ we will be able to use `list[Setup]` but that's not the case yet (waiting for v1.0 ? :p)

Reply:
gotcha. I'll modify accordingly